utils/update_checkout/update-checkout-config.json

```diff
@@ -308,7 +308,7 @@
        "tensorflow": {
             "aliases": ["tensorflow"],
             "repos": {
-                "llvm-project": "13c814f63ed68657fb480dfa428c94cbe82fc064",
+                "llvm-project": "8d389494ae739c7c14dfed9b554ebe0993014568",
```

Comment:
Won't you need to update this with the following change https://github.com/apple/llvm-project/pull/751?

Reply:
Just merged that, and updated this to the corresponding commit hash.